OneDrive-Cloud-Player/ViewModels/VideoPlayerPageViewModel.cs

```diff
@@ -205,20 +214,49 @@ public bool IsBackBtnEnabled
             }
         }
 
+        private TrackDescription _selectedSubtitleTrack;
+
+        public TrackDescription SelectedSubtitleTrack
+        {
+            get
+            {
+                return _selectedSubtitleTrack;
+            }
```

Comment:
```suggestion
            get { return _selectedSubtitleTrack; }
```
like the rest of the oneline getters?